driver/network_driver.go

```diff
@@ -20,6 +23,11 @@ import (
 	osutils "github.com/projectcalico/libnetwork-plugin/utils/os"
 )
 
+const DOCKER_LABEL_PREFIX = "org.calico.workloadendpoint.label."
```

Comment:
I know it's a little more verbose, but since we don't have the `calico.org` domain, I think it's better for the prefix to start `org.projectcalico`. It will need to be updated in a few other places - I think just the readme and the tests.

To make it less verbose, I think the `.workloadendpoint` part can be dropped. I don't think it adds much value and just makes the string longer.